### DISCUSSION TITLE
adding Helm repo

### DISCUSSION THREAD

**Discussion by kamikaze** [bank-vaults/bank-vaults#2281] [Q&A] [answered]
Hi.
Is there any way to add helm repo with `helm repo add` command? It is much easier to install/upgrade it via Lens IDE then.
Thanks

**Comment by ramizpolic** [comment, marked as answer]
Bank-Vaults uses OCI-based helm charts which do not support `helm repo` commands (check: https://github.com/helm/helm/issues/10565).
I am not sure if Lense IDE supports OCI registries yet.
